### PR TITLE
Validation form

### PR DESCRIPTION
**Pull request by TahenyBELGUITH**
I add tests to validate the form : 
I test the inputs using DOM manipulation

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @TahenyBELGUITH,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- [ ] The form shouldn't submit when the email address has uppercase letters.

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: validation.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    valid = true;
+  }
+  return valid;
+};
+
+const isEmailValid = (email) => {
+  const re =
+    /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  return re.test(email);
+};
```
Comment (mikenath223) [inline validation.js:44]: - [ ] The form still submits when the email field has uppercase letters, which shouldn't be [the case](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#:~:text=You%20need%20to%20implement,be%20in%20lower%20case.). Kindly fix it.

Comment (TahenyBELGUITH) [reply validation.js]: done

File: index.html [hunk 2]
```diff
@@ -109,6 +109,46 @@ <h4><a class="intro-connect " href="# ">LET'S CONNECT</a></h4>
...
+                <img class="card-img desk" id="img" src="images/Snapshoot.svg" alt="Tonic" />
+                <div class="desk-pop">
+                    <div class="first">
+                        <p class="text-secondary mob desk" id="description"></p>
+                    </div>
+                    <div class="second">
+                        <ul class="card-tags desk" id="tags"></ul>
+                        <div class="line"></div>
+                        <div class="popup-button desk">
+                            <a href="#" class="button btn-element desk">See Live
```
Comment (abruzy) [inline index.html:139]: _[OPTIONAL]_

- [ ] Kindly remove commented out code as this mess your workspace

File: index.html [hunk 3]
```diff
@@ -289,17 +329,40 @@ <h4 class="lang-hover">Skills</h4>
...
+                        required>Write your message Here!</textarea>
+                    <small></small>
+                </div>
+                <div class="form-field">
+                    <button class="tonic-button-submit submit-btn " type="submit">Get In touch</button>
+                </div>
+                <p class="invalid"></p>
+            </form>
+
+        </div>
```
Comment (abruzy) [inline index.html:355]: - [ ] the error message should show  up close to the `submit`
Screenshot 2021-08-06 at 16 05 42
 button

- [ ] it should be at the bottom of the textarea field

_[OPTIONAL]_

- I also some weird behavior in your form, the input field is jumping when hover, why is that, I highly recommend that you fix as well

**Review by abruzy — CHANGES_REQUESTED** [review]
Hi Team 👋🏽, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (TahenyBELGUITH) [reply index.html]: done thank you for your comment

File: index.html (hunk 3 above)
Comment (TahenyBELGUITH) [reply index.html]: done

**Review by hemant-soni-vst-au4 — APPROVED** [review]
Hi @TahenyBELGUITH, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**